app/parse.py

```diff
@@ -15,8 +27,70 @@ class Product:
     num_of_reviews: int
 
 
+def parse_single_product(product_soup: BeautifulSoup) -> Product:
```

Comment:
You can awoid using BeautifulSoup here, if you'are already using selenium  (optional)